jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/milestones/CaseMilestoneListPresenter.java

```diff
@@ -70,5 +105,7 @@ protected void refreshData(String caseId) {
         void setCaseMilestoneList(List<CaseMilestoneSummary> caseMilestoneList);
 
         CaseMilestoneSearchRequest getCaseMilestoneSearchRequest();
+
+        void hideLoadButton(boolean doHide);
```

Comment:
Seems that this is never called, button is always visible even if the number of items is lower than the page size.